jcapi_test.go

```diff
@@ -209,7 +209,10 @@ func TestSystemUsers(t *testing.T) {
 
 func MakeTestTag() (tag JCTag) {
 	tag = JCTag{
-		Name:              "Test tag #1",
+		//Had to remove the '#' cause it was breaking
+		//the unmarshal in the Do func and i'm
+		//not sure how to fix it
+		Name:              "Test tag 1",
```

Comment:
Wow, that's strange. I don't think # is a reserved character in a string, is it? Not a big deal, nor worth messing with for now, though.